src/applications/disability-benefits/2346/containers/IntroductionPage.jsx

```diff
@@ -92,7 +92,7 @@ class IntroductionPage extends React.Component {
           startText="Start the Application"
         />
         <div className="omb-info--container vads-u-padding-left--0">
-          <OMBInfo resBurden={30} ombNumber="2900-0862" expDate="02/28/2022" />
+          <OMBInfo resBurden={60} ombNumber="2900-0862" expDate="02/28/2022" />
```

Comment:
Please find out if you even need an OMBInfo section... I looked at the form - https://www.va.gov/vaforms/va/pdf/VA2346a.pdf - and don't see any OMB info. The number you have above is for form 20-0996 (Higher-Level Review), the one I'm working on.